analysed-packages/altgraph/version-0.17.4/README.md

```diff
@@ -0,0 +1,20 @@
+﻿## Download Location
+
+https://github.com/ronaldoussoren/altgraph
+
+## Package URL (purl)
+
+pkg:github/ronaldoussoren/altgraph@0.17.4
+
+## Creator
+
+Kanzlei Jun on behalf of AUDI AG
+
+## Reviewers
+
+The information was reviewed by:
+
+
+## Comment
+
+Only the Python packages of the components have been curated, not the entire packages found in the download location.
```

Comment:
The components were downloaded automatically using Python's own package management tool. The folders I sent you are complete in the sense that they contain all Python code necessary to use the Python package. However, this does not necessarily have to be all source code related to this OSS project on Github.

Reply:
Ah I see, this might become a problem, because many package management systems have an internal heuristic implemented which leads to the fact that a downloaded package has different content when downloaded at another point in time.